src/components/Article.jsx

```diff
@@ -1 +1,28 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
```

Comment:
don't forget to add types for the input arguments

Reply:
Types like in Typescript? We have just jsx files, and typescript appeared later (through two tasks), or maybe you mean smth else?